File: Homework1/zooMain.java
public class zooMain{
  public static void main(String args[]){

    Zoo newZoo = new Zoo();
    newZoo.addAnimal(new Cat("Charlie"));
    newZoo.addAnimal(new Cat("Carl"));
    newZoo.addAnimal(new Cat("Creg"));
    newZoo.addAnimal(new Lion("Larry"));
    newZoo.addAnimal(new Lion("Lissa"));
    newZoo.addAnimal(new Dog("Dennis"));
    newZoo.addAnimal(new Tiger("Tyler"));
    newZoo.addAnimal(new Wolf("Watson"));
    newZoo.addAnimal(new Hippo("Henry"));
    newZoo.addAnimal(new Elephant("Eagor"));
    newZoo.addAnimal(new Dog("David"));
    newZoo.addZookeper(new Zookeeper("Bruce",24, 250000));

    //wake up
    newZoo.wakeUp();
    //roll call
    newZoo.rollCall();
    //feed animals
    newZoo.feedAnimals();
    //excersize
    newZoo.excersizeAnimals();
    //shutdown
    newZoo.shutDownZoo();
    return ;
  }
}

File: Homework2/ZooMain.py
from Animal import *
from Zoo import *
from Zookeeper import *
#from ZooAnnouncer import *

#class zooMain:
  #def main():

Zoo = Zoo()
Zoo.addAnimal(Cat("Charlie"))
Zoo.addAnimal( Cat("Carl"))
Zoo.addAnimal( Cat("Creg"))
Zoo.addAnimal( Lion("Larry"))
Zoo.addAnimal( Lion("Lissa"))
Zoo.addAnimal( Dog("Dennis"))
Zoo.addAnimal( Tiger("Tyler"))
Zoo.addAnimal( Wolf("Watson"))
Zoo.addAnimal( Hippo("Henry"))
Zoo.addAnimal( Elephant("Eagor"))
Zoo.addAnimal( Dog("David"))
Zoo.addZookeper( Zookeeper("Bruce",24, 250000))

#wake up
Zoo.wakeUp()
#roll call
Zoo.rollCall()
#feed animals
Zoo.feedAnimals()
#excersize
Zoo.excersizeAnimals()
#shutdown
Zoo.shutDownZoo()
